### DISCUSSION TITLE
Managing providers in Dist.X

### DISCUSSION THREAD

**Discussion by pedroigor** [keycloak/keycloak#8611] [Ideas]
As a result of initial discussions at https://github.com/keycloak/keycloak/discussions/8501, we should provide the necessary means for managing providers through Dist.X CLI.

An initial proposal was submitted here https://github.com/keycloak/keycloak-community/pull/328. For now, it is covering only the query operations.

**Comment by stianst** [comment]
It could be nice to somehow align SPIs, providers, features, and selecting the "type of things" like DB vendor. See some more thoughts here:

https://github.com/keycloak/keycloak/pull/8369#issuecomment-948480246
